libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxManeuverView.kt

```diff
@@ -335,23 +357,23 @@ class MapboxManeuverView : ConstraintLayout {
      * @param roadShield shield if available otherwise null
      */
     @Deprecated(
-        message = "The method can only render one shield if an instruction has multiple shields",
+        message = "The method may or may not render multiple shields for a given instruction",
         replaceWith = ReplaceWith("renderPrimary(primary, roadShields)")
     )
     @JvmOverloads
     fun renderPrimaryManeuver(primary: PrimaryManeuver, roadShield: RoadShield? = null) {
         val shields = ifNonNull(roadShield) {
-            listOf(it)
+            setOf(it)
         }
         renderPrimary(primary, shields)
     }
 
     /**
      * Invoke the method to render primary instructions on top of [MapboxManeuverView]
      * @param primary PrimaryManeuver
-     * @param roadShields list of shields if available otherwise null
+     * @param roadShields set of shields if available otherwise null
      */
-    fun renderPrimary(primary: PrimaryManeuver, roadShields: List<RoadShield>?) {
+    fun renderPrimary(primary: PrimaryManeuver, roadShields: Set<RoadShield>?) {
```

Comment:
This and all other new `render` methods should take `RouteShieldResult` (or the wrapped shield) as an argument instead of relying on the already deprecated `RoadShield` type.